test/utils.test.js

```diff
@@ -72,4 +73,34 @@ describe('Utility Functions', () => {
     expect(putZero('1')).toBe('01');
     expect(putZero('12')).toBe('12');
   });
+
+  test('should create a range of days between to date', () => {
+    const startDate = { year: 1398, month: 6, day: 29 };
+    const endDate = { year: 1398, month: 7, day: 3 };
+    expect(createDateRangeDays(startDate, endDate, [])).toEqual([
```

Comment:
It sounds to me like multiple-day selection instead of a range! Multiple-day selection API has been proposed in #22.

Reply:
some kind of , but using range date selection to get it